### PR TITLE
feat(issues-8): Criar projeto initial Xcode

### LINKED ISSUE #8: Criar project Xcode
- Criar projeto Xcode
- Target 14
- Interface swiftUI

### PR DESCRIPTION
**Pull request by AlexandreBCardoso**
## 🎯 Objetivo

Criação de projeto initial em Xcode.

&nbsp;
## 🛠 O que mudou

Adicionado projeto Xcode.

&nbsp;
## 🧪 Como testar

N/A

&nbsp;
## 🚩 Tipo de alteração

- [x] Nova feature
- [ ] Melhorias no código
- [ ] Remoção de código
- [ ] Melhorias na infra (ci, scripts, builds, etc)
- [ ] Atualização de docs
- [ ] Crash em runtime
- [ ] Crash em buildtime
- [ ] Erro funcional
- [ ] Erro visual

&nbsp;
## 📝 Checklist

- [ ] Aplica boas práticas de codificação
- [ ] Cobertura de testes **maior** ou **igual** a anterior
- [ ] Executado SWIFTLINT e corrigido erros e warnings

### CODE REVIEW

File: .DS_Store
Comment (gustavochx) [inline .DS_Store:1]: sanity-check: Does this file need to be part of the commit? I thought the file had been mapped as part of `.gitignore`

File: JobsDeviOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata
Comment (gustavochx) [inline JobsDeviOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata:1]: I would recommend removing this file from part of your commit, sometimes your workspace setup could not work on other machines or on a CI/CD pipelines.

File: JobsDeviOS.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
Comment (gustavochx) [inline JobsDeviOS.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist:1]: same observation from `JobsDeviOS.xcodeproj/project.xcworkspace/contents.xcworkspacedata` here 🙂

File: JobsDeviOS/Resources/Assets.xcassets/.DS_Store
Comment (gustavochx) [inline JobsDeviOS/Resources/Assets.xcassets/.DS_Store:1]: sanity-check: I'm a bit confused, did you map `.DS_Store` is to be ignored by your `.gitignore` file, right?
If yes, could you remove them, please?

**Review by gustavochx — COMMENTED** [review]
A few comments to check. Feel free to we discuss about them 🙂🤘🏻

File: JobsDeviOS/Resources/Assets.xcassets/.DS_Store
Comment (AlexandreBCardoso) [reply JobsDeviOS/Resources/Assets.xcassets/.DS_Store:1]: Removido

**Review by gustavochx — APPROVED** [review]
LGTM!

**Pull request merged**